_dashboards/visualize/viz-index.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: default
+title: Visualize
+nav_order: 4
+has_children: true
+---
+
+# Visualize
+
+Data visualization is representing your data in a visual context. By visualizing your data, you translate complex, high-volume, or numerical data into a visual representation that is easier to process.
```

Comment:
This description is a little over-complicated for an intro on visualization. I would remove this entire paragraph.

Reply:
Removed and revised.